src/aiohomeconnect/model/event.py

```diff
@@ -67,6 +67,12 @@ def from_server_sent_event(cls, sse: ServerSentEvent) -> EventMessage:
 class EventKey(StrEnum):
     """Represent an event key."""
 
+    @classmethod
+    def _missing_(cls, _: object) -> EventKey:
+        """Return UNKNOWN for missing keys."""
+        return cls.UNKNOWN
```

Comment:
Please test that this works for the enums by trying to create a missing enum member.